src/services/search/search.js

```diff
@@ -39,26 +40,35 @@ async function search(context, mongo) {
 
   const $ = cheerio.load(body);
 
-  let results = [];
-  $('.result__title > a').each(function () {
-    const text = this.firstChild.data || 'Failed to get title';
-    const href = this.attribs.href.replace(/^\/\/duckduckgo.com\/l\/\?uddg=/, '').replace(/&rut=.*$/, '');
-    results.push({ text, href: decodeURIComponent(href) });
-  });
+  let results = $('.result__title > a')
+    .map((_, el) => ({
+      text: el.firstChild.data || 'Failed to get title',
+      href: decodeURIComponent(el.attribs.href.replace(/^\/\/duckduckgo.com\/l\/\?uddg=/, '').replace(/&rut=.*$/, '')),
+    }))
+    .get();
+
+  let snippets = $('.result__snippet')
+    .map((_, el) => el.children.map((x) => $.html(x)).join(''))
+    .get();
```

Comment:
so you won't forget. ini dijadiin satu aja biar ngapus iklannya sesuai.

![image](https://user-images.githubusercontent.com/7274326/131374435-0af380ea-5c7f-41d7-b817-2cfc1c509e3a.png)